src/main/java/uk/gov/justice/digital/hmpps/csr/api/dto/DetailDto.kt

```diff
@@ -0,0 +1,68 @@
+package uk.gov.justice.digital.hmpps.csr.api.dto
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
+import uk.gov.justice.digital.hmpps.csr.api.domain.ActionType
+import uk.gov.justice.digital.hmpps.csr.api.domain.ShiftType
+import uk.gov.justice.digital.hmpps.csr.api.model.Detail
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@ApiModel(description = "DetailDto")
+data class DetailDto @JsonCreator constructor(
```

Comment:
Is this different than what is being spewed out by the c# app? Do we have a record of all the changes that need to be implemented to wire these up together?